### DISCUSSION TITLE
Project #3 Q2

### DISCUSSION THREAD

**Discussion by ksk20011** [cooperrc/computational-mechanics#234] [General]
I had a question regarding question #2 on project #3, what should by drag coefficient be regarding the model?

**Comment by jackseitz** [comment]
I'm assuming you're asking what the drag coefficient should be? In the function description, it says it should be 0.18e-3 kg/m, so that's what I used throughout the project. Sorry if I misunderstood the question

**Reply by cooperrc** [reply]
Yes, drag is one of the default inputs for `rocket`. In the `simplerocket` ODE, there's no effect from drag so you could set it as 0.
